src/python-common/tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, py3, lint
```

Comment:
As python-common has no dependencies to other components, is it possible to keep py2 here?

Reply:
it's possible. but i think i put the reasons in another comment in this PR. probably we can continue the discussion out there.

Reply:
We can't treat python-common similar to ceph-volume as it's part of make check? I'd be :+1: for removing py27, **if** keeping it here would mean that we need to keep py27 in AddCephTest.cmake and install-deps.sh